wellregistry/registry/admin/monitoring_location.py

```diff
@@ -386,13 +456,25 @@ def site_id(obj):
     def get_urls(self):
         urls = super().get_urls()
         custom_urls = [
-            path('bulk_upload/', self.admin_site.admin_view(BulkUploadView.as_view()), name='bulk_upload'),
-            path('bulk_upload_template/', self.admin_site.admin_view(BulkUploadTemplateView.as_view()),
-                 name='bulk_upload_template'),
-            path('fetch_from_nwis/', self.admin_site.admin_view(FetchFromNwisView.as_view()), name='fetch_from_nwis')
+            path('bulk_upload/',
+                self.admin_site.admin_view(BulkUploadView.as_view()),
+                name='bulk_upload'),
+            path('bulk_upload_template/',
+                self.admin_site.admin_view(BulkUploadTemplateView.as_view()),
+                name='bulk_upload_template'),
+            path('fetch_from_nwis/',
+                self.admin_site.admin_view(FetchFromNwisView.as_view()),
+                name='fetch_from_nwis'),
+            path(self.custom_urls['siteno_autocomplete'],
+                self.admin_site.admin_view(SiteNoAutoCompleteView.as_view(model_admin=self)),
+                name='siteno_autocomplete')
         ]
         return custom_urls + urls
 
+    def get_site_no_search_results(self, request, queryset, search_term):
```

Comment:
Why did you put this method here? It seems to not be necessary for the implementation of the MonitoringLocationAdmin class and the method doesn't use the class. Seems like you could just add this one line of code where get_site_no_search_results is called above. And then you can remove code there because you don't need to worry about setting distinct to True.

Reply:
resolved